migrations/sql/V2020.05.05.10.42__0_permit_data_transform.sql

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
This file will be removed, comments can be dispersed to their relevant code, and file timestamps will be updated.

Reply:
This is still todo

Reply:
As this is a single migration, wont deleting this invalidate the flyway tables?